### ISSUE TITLE
感谢，在tiny目录下sh run_train.sh时报错

### ISSUE THREAD

**Issue by Kangzhiwen** [PaddlePaddle/PaddleSpeech#442]
按照readme一步步来的，然后遇到了swig_decoders不存在的问题，按照issue解决后，继续执行train报如下错误：
W0409 11:01:41.920228  2557 init.cc:209] Warning: PaddlePaddle catches a failure signal, it may not work properly
W0409 11:01:41.920295  2557 init.cc:211] You could check whether you killed PaddlePaddle thread/process accidentally or report the case to PaddlePaddle
W0409 11:01:41.920307  2557 init.cc:214] The detail failure signal is:

W0409 11:01:41.920320  2557 init.cc:217] *** Aborted at 1586401301 (unix time) try "date -d @1586401301" if you are using GNU date ***
W0409 11:01:41.923130  2557 init.cc:217] PC: @                0x0 (unknown)
run_train.sh: 行 35:  2557 段错误               (吐核)CUDA_VISIBLE_DEVICES=0,1,2,3 python -u train.py --batch_size=4 --num_epoch=20 --num_conv_layers=2 --num_rnn_layers=3 --rnn_layer_size=2048 --num_iter_print=1 --save_epoch=1 --num_samples=64 --learning_rate=1e-5 --max_duration=27.0 --min_duration=0.0 --test_off=False --use_sortagrad=True --use_gru=False --use_gpu=False --is_local=True --share_rnn_weights=True --train_manifest='data/tiny/manifest.tiny' --dev_manifest='data/tiny/manifest.tiny' --mean_std_path='data/tiny/mean_std.npz' --vocab_path='data/tiny/vocab.txt' --output_model_dir='./checkpoints/tiny' --augment_conf_path='conf/augmentation.config' --specgram_type='linear' --shuffle_method='batch_shuffle_clipped'
Failed in training!

paddlepaddle安装如下，安装的无GPU版本，非docker安装：
PaddlePaddle 1.7.1, compiled with
    with_avx: ON
    with_gpu: OFF
    with_mkl: ON
    with_mkldnn: ON
    with_python: ON

DeepSpeech也是非Docker安装。

现在解决不了这个问题，请大家指教。

**Comment by Kangzhiwen** [comment]
亲们，可能找到原因了 是因为我的服务器的CPU不支持AVX2指令集，所以导致了这个错误。
